### ISSUE TITLE
Update feature list

### ISSUE THREAD

**Issue by melroy89** [magicstone-dev/magicstone.dev#46]
One the main reasons I will migrate to Ecko is Markdown support!

You might want to add this to your feature list?

https://magicstone.dev/ecko-features/

**Comment by weex** [comment]
I agree and would merge a PR adding that. That being said Ecko is in somewhat of a stuck/dormant state right now so it makes more sense to market the problems than the features at this point.

What's more likely for the home page is for Ecko to be replaced with the next C4 flagship for Magic Stone. I'm working on something and nearing launch but it's unclear if there'll be uptake.

**Comment by melroy89** [comment]
This almost sounds like a failed fork. That would be very sad.

**Comment by weex** [comment]
It would be sad if that were the only result of the project. I'd just say it's not archived yet and even then you are free to fork it at any time.
